pdfbox/__init__.py

```diff
@@ -176,3 +176,73 @@ def extract_text(self, input_path, output_path='',
         p = sarge.capture_stdout(cmd)
         if not output_path:
             return p.stdout.text
+
+    def pdf_to_images(self, input_path, output_path, password=None,
+                      imageType=None, outputPrefix=None,
+                      startPage=None, endPage=None,
+                      page=None, dpi=None, color=None, cropbox=None,time=None):
+        """
+        Extract all text from PDF file.
```

Comment:
Isn't this function's purpose to convert a PDF to images?

Reply:
Yes, this is misleading. Will correct it.